tests/integration/test_enrollment/test_agent_auth_enrollment.py

```diff
@@ -0,0 +1,76 @@
+'''
+brief: This module verifies the correct behavior of the agent-auth enrollment tool under different configurations
+copyright:
+    Copyright (C) 2015-2021, Wazuh Inc.
+    Created by Wazuh, Inc. <info@wazuh.com>.
+    This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+metadata:
+    component:
+        - Agent
+    modules:
+        - agent-auth
+    daemons:
+        - agent-auth
+    operating_system:
+        - Ubuntu
+        - CentOS
+        - Windows
+    tiers:
+        - 0
+    tags:
+        - Enrollment
+        - Agent-auth
+'''
+
+import pytest
+
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.services import control_service
+from conftest import *
+
+# Marks
+pytestmark = [pytest.mark.linux, pytest.mark.win32, pytest.mark.tier(level=0), pytest.mark.agent]
+
+configurations = load_wazuh_configurations(configurations_path, __name__)
```

Comment:
The `configurations_path` is not defined

Reply:
Fixed